src/main/java/com/rcv/GuiConfigController.java

```diff
@@ -375,20 +500,51 @@ private void loadConfig(ContestConfig config) {
     textFieldUndervoteLabel.setText(config.getUndervoteLabel());
     textFieldUndeclaredWriteInLabel.setText(config.getUndeclaredWriteInLabel());
     textFieldRulesDescription.setText(config.getRulesDescription());
-    radioBatchEliminationTrue.setSelected(config.isBatchEliminationEnabled());
-    radioContinueUntilTwoCandidatesRemainTrue.setSelected(
-        config.willContinueUntilTwoCandidatesRemain());
-    radioExhaustOnDuplicateCandidateTrue.setSelected(config.isExhaustOnDuplicateCandidateEnabled());
-    radioTreatBlankAsUndeclaredWriteInTrue.setSelected(
-        config.isTreatBlankAsUndeclaredWriteInEnabled());
+    setToggleBoolean(
+        config.isBatchEliminationEnabled(), radioBatchEliminationTrue, radioBatchEliminationFalse);
+    setToggleBoolean(
+        config.willContinueUntilTwoCandidatesRemain(),
+        radioContinueUntilTwoCandidatesRemainTrue,
+        radioContinueUntilTwoCandidatesRemainFalse);
+    setToggleBoolean(
+        config.isExhaustOnDuplicateCandidateEnabled(),
+        radioExhaustOnDuplicateCandidateTrue,
+        radioExhaustOnDuplicateCandidateFalse);
+    setToggleBoolean(
+        config.isTreatBlankAsUndeclaredWriteInEnabled(),
+        radioTreatBlankAsUndeclaredWriteInTrue,
+        radioTreatBlankAsUndeclaredWriteInFalse);
+  }
+
+  // TODO: this function assumes that if a field isn't provided in the config, it'll be false
+  private void setToggleBoolean(boolean condition, RadioButton radioTrue, RadioButton radioFalse) {
+    if (condition) {
+      radioTrue.setSelected(true);
+    } else {
+      radioFalse.setSelected(true);
+    }
   }
 
   private boolean getToggleBoolean(ToggleGroup toggleGroup) {
     return ((RadioButton) toggleGroup.getSelectedToggle()).getText().equals("True");
   }
 
   private Integer getIntValueElse(TextField textField, Integer defaultValue) {
-    return !textField.getText().isEmpty() ? Integer.valueOf(textField.getText()) : defaultValue;
+    Integer returnValue;
+    try {
+      if (textField.getText().isEmpty()) {
+        throw new IllegalArgumentException();
+      }
+      returnValue = Integer.valueOf(textField.getText());
+    } catch (Exception exception) {
+      Logger.log(
+          Level.WARNING,
+          "Integer required! Illegal value '%s' was replaced by '%s'",
```

Comment:
If you leave a field blank, you get this: “WARNING: Integer required! Illegal value '' was replaced by 'null’”

Reply:
Yeah, I noticed this too. Will file a minor issue to handle it.